EverestSplash/EverestSplash.cs

```diff
@@ -330,43 +329,34 @@ private void LoadTextures() {
     private void HandleWindow() {
         SDL.SDL_ShowWindow(windowInfo.window);
 
-        // Animation values, SDL timers are a pain to use, this is easier
+        // Global loop vars
         int startEverestSpriteIdx = 0;
         string[] startingCelesteText = { // DO Make sure that the longest string goes first, for caching reasons
             "Starting Celeste...", "Starting Celeste", "Starting Celeste.", "Starting Celeste.."
         };
-        AnimTimer(500, () => {
-            windowInfo.startingEverestFontCache.SetText(startingCelesteText[startEverestSpriteIdx]);
-            startEverestSpriteIdx = (startEverestSpriteIdx + 1) % startingCelesteText.Length;
-        });
+        double textAccDt = 0;
+        // Set the first text
+        windowInfo.startingEverestFontCache.SetText(startingCelesteText[startEverestSpriteIdx]);
         
         int realBgH = windowInfo.bgGradientTexture.Height * WindowWidth / windowInfo.bgGradientTexture.Width;
-        int bgBloomPos = -realBgH/2;
-        AnimTimer(16, () => {
-            bgBloomPos += 1;
-            if (bgBloomPos > realBgH/2) {
-                bgBloomPos = -realBgH/2;
-            }
-        });
+        double bgBloomPos = -realBgH/2f;
```

Comment:
```suggestion
        double bgBloomPos = -realBgH/2d;
```
The division will be done in a 32-bit float because of the float literal, losing precision. Use a double literal instead.